backend/registration-service/pom.xml

```diff
@@ -48,6 +48,10 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+    <dependency>
+      <groupId>org.liquibase</groupId>
+      <artifactId>liquibase-core</artifactId>
+    </dependency>
```

Comment:
Line these up in the XML if you can - a bit pedantic but it'll read better! Also, where does the version come from, just as ideally it should be pinned.